person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_writer :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
```

Comment:
- [ ] Kindly notice that in the `of_age?` method you should check if the `age` is greater than or equal to 18 as required.

```ruby
def of_age?
    @age >= 18
end
```